lxd/instance/drivers/driver_qemu.go

```diff
@@ -1017,6 +1017,11 @@ func (d *qemu) Start(stateful bool) error {
 		return err
 	}
 
+	// Ensure secureboot is turned off for images that are not secureboot enabled
+	if d.localConfig["image.requirements.secureboot"] == "false" && shared.IsTrue(d.expandedConfig["security.secureboot"]) {
```

Comment:
An empty `security.secureboot` means that it's enabled, so you need to check for empty too.

Reply:
So something like `(d.expandedConfig["security.secureboot"] == "" || shared.IsTrue(d.expandedConfig["security.secureboot"]))` should do it